pyscriptjs/src/components/base.ts

```diff
@@ -204,8 +195,8 @@ export class BaseEvalElement extends HTMLElement {
     } // end eval
 
     runAfterRuntimeInitialized(callback: () => Promise<void>){
-        pyodideLoaded.subscribe(value => {
-            if ('runPythonAsync' in value) {
+        runtimeLoaded.subscribe(value => {
+            if ('run' in value) {
```

Comment:
I don't think we need this if here anymore... do we?

Reply:
Oh, nevermind... we do (for now). This is very specific to pyodide so it'd be good to move this custom logic to check if the runtime is "ready" to the runtime abstraction itself. Like adding a `isReady` method that each runtime needs to implement.

Reply:
will do it in follow up PR if it's okay?